pagopa/io-app
# fix: [IA-89] Missing accessibility role on share data links

## Short description
Added accessibility role on share data links, if screen reader is enabled I added the possibility to press the Body text component, so the user knows there is a link inside it.

## How to test
1. With the screen reader activated go to profile -> privacy and conditions of use -> share data.
2. Tap all links and check if the user can understand that they are links.
3. For the link "the privacy policy and the terms and conditions of use" the whole block of text should be read and behave like a link.


https://user-images.githubusercontent.com/61834253/124151619-08588a00-da93-11eb-8b29-014f2a34a71e.mp4
